cpp/dolfinx/io/ADIOS2_utils.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2021-2023 Jørgen S. Dokken and Garth N. Wells
```

Comment:
@ampdes You should add your own name here and update copyright year:)

Reply:
Done.